pkgs/tools/text/crowdin-cli/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchurl
+, gawk
+, git
+, gnugrep
+, jre
+, unzip
+}:
+
+let
+  pname = "crowdin-cli";
+  version = "3.6.4";
+in stdenv.mkDerivation {
+  inherit pname version;
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  pname = "crowdin-cli";
  version = "3.6.4";
```

Reply:
done. I usually avoid `rec` expression and follow recommendations from https://nix.dev/anti-patterns/language#rec-expression. Why do you prefer using `rec`?

Reply:
That doesn't apply to mkDerivation really. It is more useful when writing library functions.